webapp/locales/en.json

```diff
@@ -632,8 +637,12 @@
     "deleteUserAccount": {
       "accountDescription": "Be aware that your Posts and Comments are important to our community. If you still choose to delete them, you have to mark them below.",
       "accountWarning": "You CAN'T MANAGE and CAN'T RECOVER your Account, Posts, or Comments after deleting your account!",
+      "accountWarningAdmin": "The account, contributions or comments can NOT BE ADMINISTERED OR RESTORED after deletion!",
+      "accountWarningIsAdmin": "Attention! You are about to delete a user account!",
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-08-26T13:53:04Z" title="Wednesday, August 26th 2020, 3:53:04 pm +02:00">Aug 26, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -632,8 +637,12 @@
     "deleteUserAccount": {
       "accountDescription": "Be aware that your Posts and Comments are important to our community. If you still choose to delete them, you have to mark them below.",
       "accountWarning": "You CAN'T MANAGE and CAN'T RECOVER your Account, Posts, or Comments after deleting your account!",
+      "accountWarningAdmin": "The account, contributions or comments can NOT BE ADMINISTERED OR RESTORED after deletion!",
+      "accountWarningIsAdmin": "Heads up! You are about to delete a user account!",
```


```suggestion
      "accountWarningIsAdmin": "Attention! You are about to delete a user account!",
```

`Attention!` is more appropriate …